pkgs/applications/networking/aws-client-vpn/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, lib
+, curl
+, dpkg
+, fetchurl
+, gtk3
+, icu
+, kerberos
+, libredirect
+, lttng-ust
+, wrapGAppsHook
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  name = "aws-client-vpn-${version}";
+  version = "1.0.0";
+
+  src = fetchurl {
+    url = "https://d20adtppz83p9s.cloudfront.net/GTK/${version}/awsvpnclient_amd64.deb";
+    sha256 = "0nxmx83wbzgx648fcw5rqbdzi6v93qkpcnbb6qf1z6zzpg17c4br";
+  };
+
+  nativeBuildInputs = [ dpkg wrapGAppsHook ];
+
+  unpackPhase = ''
+    dpkg -x $src .
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    cp -av etc opt/awsvpnclient usr/share $out
+    mkdir -p $out/bin
+    ln -s "$out/awsvpnclient/AWS VPN Client" "$out/bin/AWS VPN Client"
+    ln -s "$out/awsvpnclient/Service/ACVC.GTK.Service" "$out/bin/ACVC.GTK.Service"
```

Comment:
We cannot have $out/awsvpnclient

Reply:
I wasn't sure what to do here - the package is meant to be installed in `/opt` and all executables and libraries are under `/opt/awsvpnclient` in the package. Would it be better to just copy `opt` to `$out/opt`? (I think the binaries hardcode the `/opt/awsclient` path, which is what made the redirects necessary, but it seems like we'd need to preserve the structure).

Reply:
``/opt/awsvpnclient`` wont work anyway and if we change the path anyway we can also rewrite it to $out/lib etc.

Reply:
Thanks, I updated to put `awsvpnclient` under `lib/awsvpnclient` in 48375ff